x-pack/plugins/canvas/tasks/helpers/webpack.plugins.js

```diff
@@ -64,63 +69,14 @@ module.exports = {
 
   module: {
     rules: [
-      // There's some React 15 propTypes funny business in EUI, this strips out propTypes and fixes it
-      {
-        test: /(@elastic[\/\\]eui|moment)[\/\\].*\.js$/,
-        loaders: 'babel-loader',
-        options: {
-          babelrc: false,
-          presets: [
-            'react',
-            [
-              'env',
-              {
-                targets: {
-                  node: 'current',
-                },
-              },
-            ],
-          ],
-          plugins: [
-            'transform-react-remove-prop-types', // specifically this, strips out propTypes
-            'pegjs-inline-precompile',
-            'transform-object-rest-spread',
-            'transform-async-to-generator',
-            'transform-class-properties',
-            [
-              'inline-react-svg',
-              {
-                ignorePattern: 'images/*',
-                svgo: {
-                  plugins: [{ cleanupIDs: false }, { removeViewBox: false }],
-                },
-              },
-            ],
-          ],
-        },
-      },
       {
         test: /\.js$/,
+        exclude: [/node_modules/],
```

Comment:
Doesn't this stop webpack from bundling node modules? We need that to happen for all environments other than server. All the plugins need to be completely static.

Reply:
Nope, just keeps it from transforming the node_modules with babel, and was there before but wasn't next to the `test:` property, which feels like a logical grouping worth keeping.

Reply:
Ah, I missed the removed line below. 👍 

Not part of this PR then, but do you think we should be transforming `node_modules` code? Seems like it would be safer, since we need this code to run in browsers. That's why we were specifically handling EUI and moment previously.

Reply:
> Do you think we should be transforming node_modules code? 
I don't, and we haven't ever done that in Kibana. I think it does require seeing if a module supports running in the browser and which browsers it supports, but that strategy has worked out well for us (and probably every community config I've ever seen).